### ISSUE TITLE
Security

### ISSUE THREAD

**Issue by tbodt** [romgrk/vim-exeline#1]
If I installed this plugin, opening an attacker controlled file in Vim could take over my computer. As far as I understand there's no mitigation...

**Comment by romgrk** [comment]
Yes. I'm aware of it, but this plugin is so unused that I never bothered doing anything. I'll add a warning to the readme at the moment.

One possible mitigation would be to do something like [vim-addon-local-vimrc](https://github.com/MarcWeber/vim-addon-local-vimrc) does, which prompts the user the first time a file is going to be executed.

**Comment by leighmcculloch** [comment]
Prompting a user the first time it gets used will only go so far. It doesn't take into account if you pull a later version down and the code has changed, but not by you. Realistically the cache for when to prompt or not would need to be based on file contents, and not just filename, but that would defeat the purpose. Everytime you make a change it'd change. Unless you could differentiate between the vim buffer making the change, and then not reprompting, and the file changing on disk.

**Comment by romgrk** [comment]
Fair point, that also applies to vim-addon-local-vimrc if a local vimrc is included in the git repo.

I recommend anyone concerned about security to not use this plugin.
